### PR TITLE
vendor: assorted vendor updates for containerd 1.7.0

### PR DESCRIPTION
**Pull request by thaJeztah**
- relates to https://github.com/moby/moby/pull/44530

Most (all?) of these are probably safe to take, or even backport to 23.0 if we want to.

### vendor: golang.org/x/sys v0.6.0

full diff: https://github.com/golang/sys/compare/v0.5.0...v0.6.0

### vendor: github.com/imdario/mergo v0.3.13

full diff: https://github.com/imdario/mergo/compare/v0.3.12...v0.3.13

### vendor: github.com/opencontainers/selinux v1.11.0

full diff: https://github.com/opencontainers/selinux/compare/v1.10.2...v1.11.0

### vendor:	github.com/coreos/go-systemd/v22 v22.5.0

full diff: https://github.com/coreos/go-systemd/compare/v22.4.0...v22.5.0

### vendor: github.com/godbus/dbus/v5 v5.1.0

full diff: https://github.com/godbus/dbus/compare/v5.0.6...v5.1.0

### vendor: github.com/klauspost/compress v1.16.3

full diff: https://github.com/klauspost/compress/compare/v1.15.12...v1.16.3

### vendor: github.com/opencontainers/runc v1.1.4

no changes in vendored files

full diff: https://github.com/opencontainers/runc/compare/v1.1.3...v1.1.4

### vendor: github.com/opencontainers/runtime-spec v1.1.0-rc.1

full diff: https://github.com/opencontainers/runtime-spec/compare/1c3f411f0417...v1.1.0-rc.1

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]

**Comment by thaJeztah** [comment]
@vvoland @corhere ptal 🤗

File: vendor.mod [hunk 1]
```diff
@@ -53,7 +53,7 @@ require (
 	github.com/hashicorp/serf v0.8.5
 	github.com/imdario/mergo v0.3.13
 	github.com/ishidawataru/sctp v0.0.0-20210707070123-9a39160e9062
-	github.com/klauspost/compress v1.15.12
+	github.com/klauspost/compress v1.16.3
```
Comment (corhere) [inline vendor.mod:56]: (File comments are great, now we just need commit comments.)

Your commit message for vendoring klauspost/compress should have https://github.com/klauspost/compress/compare/v1.15.12...v1.16.3 as the diff link.

**Review by corhere — APPROVED** [review]
One change caught my eye which I think is worth calling out: https://github.com/godbus/dbus/commit/230c6f285247d49abc850f47248f45f3c60defa5

The `static_build` build tag no longer has any meaning to the dbus module, though I don't believe we need to take any action. Our build scripts set both `osusergo` and `static_build` already, and it doesn't look like we can get rid of the `static_build` tag as it is still used in build constraints in a few of our own packages.

File: vendor.mod (hunk 1 above)
Comment (thaJeztah) [reply vendor.mod:56]: DOH! Fixed 👍

**Comment by thaJeztah** [comment]
> One change caught my eye which I think is worth calling out: [godbus/dbus@230c6f2](https://github.com/godbus/dbus/commit/230c6f285247d49abc850f47248f45f3c60defa5)
> 
> The `static_build` build tag no longer has any meaning to the dbus module, though I don't believe we need to take any action. Our build scripts set both `osusergo` and `static_build` already, and it doesn't look like we can get rid of the `static_build` tag as it is still used in build constraints in a few of our own packages.

Yes, I noticed that change as well

**Pull request merged**